### ISSUE TITLE
Toggle allowing XP globes to render over XP drops

### ISSUE THREAD

**Issue by Centralizer** [runelite/runelite#4220]
As (vanilla) XP drops rise to the top of the screen, they show over the XP globes.  It would be nice to be able to have XP drops go behind the globes. This is the behavior on the vanilla RS3 client and Konduit, for instance.

**Comment by deathbeam** [comment]
I dont think this should be toggle, probably just make the requested behaviour default?
